packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -2,9 +2,15 @@ const chalk = require('chalk');
 const { core, commands } = require('../utils/cli-flags');
 const commandLineUsage = require('command-line-usage');
 
+const [, , ...rawArgs] = process.argv;
+const commandsUsed = rawArgs.filter((val) => commands.find(({ name }) => name === val));
+const flagsUsed = rawArgs.filter((val) => core.find(({ name }) => name === val.slice(2)));
+const argsUsed = [...commandsUsed, ...flagsUsed];
+const invalidArgs = rawArgs.filter((e) => !argsUsed.includes(e) && !e.includes('--color') && e !== 'version' && e !== 'help' && e !== '-v');
```

Comment:
Avoid it, we do not need to run this code every time on loading module, it is decrease performance, place it in places where used

Reply:
this code will run only when `help` or `version` is passed by the user.

Reply:
No, we already execute this code when use `require`, put it on places where you need it